src/main/java/com/thevoxelbox/voxelsniper/brush/FlatOceanBrush.java

```diff
@@ -31,9 +31,11 @@ public FlatOceanBrush() {
     private void flatOcean(final Chunk chunk) {
         for (int x = 0; x < CHUNK_SIZE; x++) {
             for (int z = 0; z < CHUNK_SIZE; z++) {
-                for (int y = 0; y < chunk.getWorld().getMaxHeight(); y++) {
-                    if (y <= this.floorLevel) {
-                        chunk.getBlock(x, y, z).setType(Material.DIRT);
+                for (int y = this.getWorld().getMinHeight(); y < chunk.getWorld().getMaxHeight(); y++) {
+                    if (y < this.floorLevel) {
```

Comment:
This seems to be changing behaviour. Previously all `y <= floorLevel` would be set to dirt.